### PR TITLE
Alert the user if nobody's around to hear a bwoink

### PR DESCRIPTION
**Pull request by 20kdc**
## About the PR

As described.

**Screenshots**
image
(Example scenario. No bars were harmed in the making of this screenshot.)

**Changelog**

:cl:
- add: You are now alerted if an AHelp is not received by anyone.

### CODE REVIEW

**Comment by ike709** [comment]
Why not just make a copy of the OOC discord bridge to forward ahelps to a discord channel?

**Comment by 20kdc** [comment]
That would require me to patch the MoMMI bot.

**Comment by PJB3005** [comment]
MoMMI has a general purpose game message forwarding system that we use on /vg/station. We can use that instead.

I'm kind of on the fence about it because I'd *rather* use Discord webhooks but MoMMI is a more centralized place to put the configuration and such so...

File: Content.Server/Administration/BwoinkSystem.cs [hunk 1]
```diff
@@ -28,7 +29,8 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
 
             // TODO: Sanitize text?
             // Confirm that this person is actually allowed to send a message here.
-            if ((senderSession.UserId != message.ChannelId) && (_adminManager.GetAdminData(senderSession) == null))
+            var senderPersonalChannel = senderSession.UserId == message.ChannelId;
+            if (senderPersonalChannel && (_adminManager.GetAdminData(senderSession) == null))
```
Comment (PaulRitter) [inline Content.Server/Administration/BwoinkSystem.cs:33]: mfw i try to ahelp but i can't type

**Review by PaulRitter — CHANGES_REQUESTED** [review]

**Comment by PaulRitter** [comment]
> 
> 
> MoMMI has a general purpose game message forwarding system that we use for adminhelps on /vg/station. We can use that instead.
> 
> I'm kind of on the fence about it because I'd _rather_ use Discord webhooks but MoMMI is a more centralized place to put the configuration and such so...

do it on MoMMI then no balls

File: Content.Server/Administration/BwoinkSystem.cs (hunk 1 above)
Comment (20kdc) [reply Content.Server/Administration/BwoinkSystem.cs:33]: ...did I seriously manage to screw up this condition
oh dear goodness
nice catch, thank you

**Comment by 20kdc** [comment]
Rebased because of .NET 6, doing testing now

**Comment by 20kdc** [comment]
Tested, things shouldn't be broken anymore

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**